src/components/EditorsDemo/EditorsDemo.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+const EditorsDemo = () => {
+  useEffect(() => {
+    window['editors-webapp-configuration'] = {
+      basename: undefined,
+      appElementId: 'doppler-editors-webapp',
+    };
+    new window.AssetServices().load(
+      'https://cdn.fromdoppler.com/editors-webapp/asset-manifest-v1.json',
```

Comment:
I know that it is only a POC, but, in a real scenario, this URL should be changed based on the environment:

* INT: https://cdn.fromdoppler.com/editors-webapp/asset-manifest-INT.json
* QA: https://cdn.fromdoppler.com/editors-webapp/asset-manifest-main.json
* Production: https://cdn.fromdoppler.com/editors-webapp/asset-manifest-v1.json